src/main/java/komo/fraczek/toukparking/service/DummyCurrencyRateProviderService.java

```diff
@@ -0,0 +1,15 @@
+package komo.fraczek.toukparking.service;
+
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+
+@Component
+public class DummyCurrencyRateProviderService implements CurrencyRateProviderService {
+
+    @Override
+    public BigDecimal getCurrencyRate(String givenCurrency) {
+        return givenCurrency.equals("PLN") ? BigDecimal.ONE : BigDecimal.ONE;
```

Comment:
trochę dziwna ta metoda skoro tak czy siak zawsze zwróci to samo...

Reply:
Inna zagwozdka: co będzie, jeżeli będziemy chcieli obsłużyć trzy (lub więcej) różne waluty?